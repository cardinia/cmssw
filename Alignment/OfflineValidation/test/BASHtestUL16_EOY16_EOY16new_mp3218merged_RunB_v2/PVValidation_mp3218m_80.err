26-Sep-2019 12:40:15 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/80000/4E8F4930-8C09-E911-A14A-EC0D9A0B3370.root
26-Sep-2019 12:40:19 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/80000/4E8F4930-8C09-E911-A14A-EC0D9A0B3370.root
Begin processing the 1st record. Run 274336, Event 912367, LumiSection 1 on stream 0 at 26-Sep-2019 12:40:26.992 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 12:43:45 CEST  Run: 274336 Event: 1201486
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 11443
%MSG
Begin processing the 1001st record. Run 274336, Event 425298, LumiSection 1 on stream 0 at 26-Sep-2019 12:45:03.850 CEST
Begin processing the 2001st record. Run 274336, Event 1665044, LumiSection 2 on stream 0 at 26-Sep-2019 12:49:04.725 CEST
Begin processing the 3001st record. Run 274336, Event 2754720, LumiSection 2 on stream 0 at 26-Sep-2019 12:52:57.734 CEST
Begin processing the 4001st record. Run 274336, Event 3733958, LumiSection 3 on stream 0 at 26-Sep-2019 12:56:57.021 CEST
Begin processing the 5001st record. Run 274336, Event 5541068, LumiSection 4 on stream 0 at 26-Sep-2019 13:00:44.935 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:01:41 CEST  Run: 274336 Event: 5929799
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1315
%MSG
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:02:29 CEST  Run: 274336 Event: 5451906
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 16383
%MSG
Begin processing the 6001st record. Run 274336, Event 5177790, LumiSection 4 on stream 0 at 26-Sep-2019 13:04:18.828 CEST
Begin processing the 7001st record. Run 274336, Event 6986419, LumiSection 5 on stream 0 at 26-Sep-2019 13:08:04.736 CEST
Begin processing the 8001st record. Run 274336, Event 7374380, LumiSection 5 on stream 0 at 26-Sep-2019 13:11:50.080 CEST
Begin processing the 9001st record. Run 274336, Event 8277491, LumiSection 6 on stream 0 at 26-Sep-2019 13:15:51.219 CEST
Begin processing the 10001st record. Run 274336, Event 9301426, LumiSection 7 on stream 0 at 26-Sep-2019 13:19:34.889 CEST
Begin processing the 11001st record. Run 274336, Event 11390539, LumiSection 8 on stream 0 at 26-Sep-2019 13:23:19.755 CEST
Begin processing the 12001st record. Run 274336, Event 11714581, LumiSection 8 on stream 0 at 26-Sep-2019 13:26:59.460 CEST
Begin processing the 13001st record. Run 274336, Event 12300794, LumiSection 9 on stream 0 at 26-Sep-2019 13:30:41.088 CEST
Begin processing the 14001st record. Run 274336, Event 15050020, LumiSection 10 on stream 0 at 26-Sep-2019 13:34:21.561 CEST
Begin processing the 15001st record. Run 274336, Event 14631706, LumiSection 10 on stream 0 at 26-Sep-2019 13:38:14.671 CEST
Begin processing the 16001st record. Run 274336, Event 16126970, LumiSection 11 on stream 0 at 26-Sep-2019 13:42:18.187 CEST
Begin processing the 17001st record. Run 274336, Event 15783058, LumiSection 11 on stream 0 at 26-Sep-2019 13:46:32.834 CEST
Begin processing the 18001st record. Run 274336, Event 18543893, LumiSection 12 on stream 0 at 26-Sep-2019 13:50:25.895 CEST
Begin processing the 19001st record. Run 274336, Event 19168068, LumiSection 13 on stream 0 at 26-Sep-2019 13:54:16.628 CEST
Begin processing the 20001st record. Run 274336, Event 20184194, LumiSection 13 on stream 0 at 26-Sep-2019 13:58:03.401 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 26-Sep-2019 13:59:57 CEST  Run: 274336 Event: 20454198
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 775
%MSG
Begin processing the 21001st record. Run 274337, Event 849742, LumiSection 3 on stream 0 at 26-Sep-2019 14:00:11.986 CEST
Begin processing the 22001st record. Run 274337, Event 1829899, LumiSection 4 on stream 0 at 26-Sep-2019 14:00:13.952 CEST
Begin processing the 23001st record. Run 274337, Event 2497118, LumiSection 4 on stream 0 at 26-Sep-2019 14:00:15.780 CEST
Begin processing the 24001st record. Run 274337, Event 2897770, LumiSection 5 on stream 0 at 26-Sep-2019 14:00:17.435 CEST
Begin processing the 25001st record. Run 274337, Event 4300995, LumiSection 6 on stream 0 at 26-Sep-2019 14:00:19.204 CEST
Begin processing the 26001st record. Run 274337, Event 4283765, LumiSection 6 on stream 0 at 26-Sep-2019 14:00:20.970 CEST
Begin processing the 27001st record. Run 274337, Event 6660480, LumiSection 7 on stream 0 at 26-Sep-2019 14:00:22.566 CEST
Begin processing the 28001st record. Run 274337, Event 6933432, LumiSection 8 on stream 0 at 26-Sep-2019 14:00:24.522 CEST
Begin processing the 29001st record. Run 274337, Event 9291817, LumiSection 9 on stream 0 at 26-Sep-2019 14:00:26.237 CEST
Begin processing the 30001st record. Run 274337, Event 8543291, LumiSection 9 on stream 0 at 26-Sep-2019 14:00:28.422 CEST
Begin processing the 31001st record. Run 274337, Event 9972413, LumiSection 10 on stream 0 at 26-Sep-2019 14:00:30.422 CEST
Begin processing the 32001st record. Run 274337, Event 11882597, LumiSection 11 on stream 0 at 26-Sep-2019 14:00:32.067 CEST
Begin processing the 33001st record. Run 274337, Event 12570514, LumiSection 11 on stream 0 at 26-Sep-2019 14:00:33.830 CEST
Begin processing the 34001st record. Run 274337, Event 14151304, LumiSection 12 on stream 0 at 26-Sep-2019 14:00:35.580 CEST
Begin processing the 35001st record. Run 274337, Event 14425170, LumiSection 13 on stream 0 at 26-Sep-2019 14:00:37.591 CEST
Begin processing the 36001st record. Run 274337, Event 14587675, LumiSection 13 on stream 0 at 26-Sep-2019 14:00:39.991 CEST
Begin processing the 37001st record. Run 274337, Event 15761252, LumiSection 14 on stream 0 at 26-Sep-2019 14:00:41.794 CEST
Begin processing the 38001st record. Run 274337, Event 17703561, LumiSection 15 on stream 0 at 26-Sep-2019 14:00:43.695 CEST
Begin processing the 39001st record. Run 274337, Event 19357521, LumiSection 16 on stream 0 at 26-Sep-2019 14:00:45.638 CEST
Begin processing the 40001st record. Run 274337, Event 18857387, LumiSection 16 on stream 0 at 26-Sep-2019 14:00:47.472 CEST
26-Sep-2019 14:00:49 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016B/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018_ver2-v1/80000/4E8F4930-8C09-E911-A14A-EC0D9A0B3370.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 OutOfBounds          -w TrackRefitter:Fi                       4        4
    2 fileAction           -s file_close                             1        1
    3 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 OutOfBounds          274336/1201486   274336/5929799   274336/20454198
    2 fileAction           PostGlobalEndRun                  
    3 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
Warning                 4                   4
System                  3                   3

dropped waiting message count 0
[9.114MB/9.114MB][100%][==================================================][9.114MB/s]  [9.114MB/9.114MB][100%][==================================================][9.114MB/s]  
