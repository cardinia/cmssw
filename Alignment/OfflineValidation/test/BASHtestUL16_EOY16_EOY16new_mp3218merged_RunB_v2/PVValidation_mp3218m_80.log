000 (3189526.080.000) 09/26 11:22:16 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189526.080.000) 09/26 12:39:52 Started transferring input files
	Transferring to host: <188.184.183.159:9618?addrs=188.184.183.159-9618+[2001-1458-301-2b--100-197]-9618&noUDP&sock=5127_2f6a_12274>
...
040 (3189526.080.000) 09/26 12:39:52 Finished transferring input files
...
001 (3189526.080.000) 09/26 12:39:52 Job executing on host: <188.184.183.159:9618?addrs=188.184.183.159-9618+[2001-1458-301-2b--100-197]-9618&noUDP&sock=3302_79cf_3>
...
006 (3189526.080.000) 09/26 12:40:01 Image size of job updated: 41992
	27  -  MemoryUsage of job (MB)
	27292  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 12:45:01 Image size of job updated: 656824
	528  -  MemoryUsage of job (MB)
	539812  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 13:00:02 Image size of job updated: 657384
	528  -  MemoryUsage of job (MB)
	540348  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 13:10:03 Image size of job updated: 658516
	529  -  MemoryUsage of job (MB)
	541444  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 13:25:05 Image size of job updated: 659120
	530  -  MemoryUsage of job (MB)
	542048  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 13:30:06 Image size of job updated: 659180
	530  -  MemoryUsage of job (MB)
	542108  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 13:35:06 Image size of job updated: 660392
	531  -  MemoryUsage of job (MB)
	543320  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 13:40:06 Image size of job updated: 660796
	531  -  MemoryUsage of job (MB)
	543724  -  ResidentSetSize of job (KB)
...
006 (3189526.080.000) 09/26 14:01:13 Image size of job updated: 1386784
	531  -  MemoryUsage of job (MB)
	543724  -  ResidentSetSize of job (KB)
...
040 (3189526.080.000) 09/26 14:01:13 Started transferring output files
...
040 (3189526.080.000) 09/26 14:01:13 Finished transferring output files
...
005 (3189526.080.000) 09/26 14:01:13 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 01:18:33, Sys 0 00:00:18  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 01:18:33, Sys 0 00:00:18  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	8790  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	8790  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     9379        1   2960796 
	   Memory (MB)          :      531     2000      2000 
...
