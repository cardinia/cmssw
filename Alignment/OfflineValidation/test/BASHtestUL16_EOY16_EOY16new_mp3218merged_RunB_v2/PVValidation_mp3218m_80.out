Job started at  Thu Sep 26 12:39:53 CEST 2019
batch dir: /pool/condor/dir_10973 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_274336_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 9.2M -rw-r--r--. 1 tkello zh 9.2M Sep 26 14:01 PVValidation_mp3218m_274336.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:39 PVValidation_mp3218m_274336_cfg.py -rw-r--r--. 1 tkello zh 7.4K Sep 26 14:01 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:40 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 26 12:39 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:40 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:39 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:39 var
Job ended at  Thu Sep 26 14:01:13 CEST 2019
